akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/audiocaptions/UpdateTranscriptPubMsgHdlr.scala

```diff
@@ -12,6 +13,8 @@ trait UpdateTranscriptPubMsgHdlr {
     val meetingId = liveMeeting.props.meetingProp.intId
 
     def broadcastEvent(userId: String, transcriptId: String, transcript: String, locale: String, result: Boolean): Unit = {
+      AudioCaptionDAO.insertOrUpdateAudioCaption(transcriptId, meetingId, transcript)
```

Comment:
It's good to do the insert at the same moment of the `broadcastEvent()`.
But just to follow the pattern of the previous migrations, I suggest to move this line to outside of the function.